test/sanity.test.js

```diff
@@ -3,23 +3,14 @@ const { expect } = require('chai');
 const { loadFixture, mine } = require('@nomicfoundation/hardhat-network-helpers');
 
 async function fixture() {
-  const signers = await ethers.getSigners();
-  const addresses = await Promise.all(signers.map(s => s.getAddress()));
-  return { signers, addresses };
+  return {};
 }
```

Comment:
Shall we remove the fixture whatsoever?

Reply:
If we do so, the sanity checkpointing check fails.

Reply:
Which is to be expected, because there is no native checkpoint system anymore, we rely on fixture for that.

So basically this file checks that the fixture actually resets the blockchain state.

Reply:
The sanity checkpointing check?

I'm not sure what that means, but, shouln't it be fine if we also remove the `Object.assign(this,...)`?

Reply:
> So basically this file checks that the fixture actually resets the blockchain state

Ah, this makes sense. Let's keep it then.